webpack.config.js

```diff
@@ -57,6 +57,7 @@ switch (buildType) {
             devServer: {
                 contentBase: "./dist",
                 port: 3000,
+                host: "0.0.0.0",
```

Comment:
never understood why this is 0.0.0.0 ?

Reply:
This will allow the webpack-dev-server to appear on network. You would do that if you want to test the UI on a mobile device in the network